bin/wazo-dist-upgrade-bullseye-check

```diff
@@ -116,6 +119,15 @@ def check_wizard_has_run():
         raise ConfdError()
 
 
+def check_gnome_not_installed():
+    command = ['dpkg-query', '--show', '--showformat', "'${Status}'", 'gnome-core']
```

Comment:
Oh, thanks @fblackburn1, that's much better.
Are the extra single quotes required though? If you leave them out the output seems to be directly the string "install ok installed" instead of having the extra single quotes.